root/etc/systemd/system/nm-prepare.service

```diff
@@ -5,6 +5,7 @@ Before=NetworkManager.target network-online.target
 
 [Service]
 Type=oneshot
+RemainAfterExit=yes
```

Comment:
This is cosmetic thing, we have the same for all the other `oneshot` services. AFAIK this only affects how systemd reports this service.